packages/wallet/src/redux/channelState/challenging/reducer.ts

```diff
@@ -64,7 +64,14 @@ const challengeTransactionFailedReducer = (
       );
       return {
         state: states.waitForChallengeInitiation(state),
-        sideEffects: { transactionOutbox: { transactionRequest, channelId: state.channelId } },
+        // TODO: This will be factored out as channel reducers should not be sending transactions itself
+        sideEffects: {
+          transactionOutbox: {
+            transactionRequest,
+            channelId: state.channelId,
+            process: 'DirectFunding',
```

Comment:
(Name this too)

Reply:
See above.

Reply:
Not applicable with the string enum changes.